browser/resources/settings/brave_overrides/a11y_page.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { RegisterPolymerTemplateModifications } from 'chrome://resources/brave/polymer_overriding.js'
+
+RegisterPolymerTemplateModifications({
+  'settings-a11y-page': (templateContent) => {
+    // We remove all the templates related to captions - there are two modes
+    // depending on whether `captionSettingsOpensExternally_` is set. We remove
+    // both.
+    const captionsLinkTemplates = templateContent.querySelectorAll('template[if*="captionSettingsOpensExternally_"]')
```

Comment:
Please error if they're not found, in lieu of a test

Reply:
Added an error if we don't find exactly 3 `captionSettingsOpensExternally` sections.